README.md

```diff
@@ -89,3 +89,17 @@ Remove a container
 ```
 docker rm container_name_or_ID
 ```
+
+## Storybook
+
+Install dependencies
+
+```
+pmpm i
```

Comment:
```suggestion
pnpm i
```

Reply:
This was my typo. TIA Richard!

Reply:
https://www.youtube.com/watch?v=rIt8WBgqOEU&ab_channel=RichardChoi